FF1Lib/Flags.cs

```diff
@@ -74,7 +74,7 @@ public class Flags : ITreasureShuffleFlags, IIncentiveFlags
 
 		public bool MapConeriaDwarves { get; set; }
 		public bool MapVolcanoIceRiver { get; set; }
+		public bool MapTitansTrove { get; set; }

```

Comment:
You could tweak this to always false too - whatever. Or default it to false in the UI.